tensorflow/lite/python/interpreter.py

```diff
@@ -292,3 +292,14 @@ def invoke(self):
 
   def reset_all_variables(self):
     return self._interpreter.ResetVariableTensors()
+
+  def set_num_threads(self, i):
+    """Set number of threads used by TFLite kernels.
+
+    If not set, kernels are running single-threaded. Note that currently,
```

Comment:
"If not set, kernels will run with a default threading configuration, which may be single-threaded"

We actually multi-thread float models by default.

Reply:
Good to know this. Thanks.

Reply:
Let's remove this API since we have the constructor path.